app/ml.py

```diff
@@ -270,7 +271,7 @@ async def get_walk_scr(city: str):
 @router.get('/city_scr/{city}')
 async def get_city_scr(city: str, crime: int=0, walk: int=0, air: int=0, rent: int=0):
     """
-    city_scr returns an overall city quality of life score (1-5)
+    city_scr returns an overall city quality of life score (1.0-5.0)
```

Comment:
Would it be possible to get a score of zero if the `calc_wghtd_city_score` rounds the result and could possibly round down to zero?

Reply:
@PalmerTurley34 hey... I updated the helper function so that the score "floor" is 1.0 (rather than 0.0).  So if the weighted average calculation happens to fall lower than 1.0, the score returned will be 1.0

See commit: https://github.com/Lambda-School-Labs/cityspire-d-ds/pull/21/commits/4fe17b24ef5b3d856865020e8334ecf21020d63b